recipes/raxml-ng/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "raxml-ng" %}
+{% set version = "0.9.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/amkozlov/{{ name|lower }}.git
+  git_rev: {{ version }}
+  sha256: 43b95ee1bddae0daee84644e9ee760a77f28bb16e2071e95cca79a30f39373e5
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - gmp
+    - bison
+    - flex
+    - cmake
```

Comment:
cmake, bison, and I think flex can be moved to the `build:` section.

Reply:
yes, flex also

Reply:
Thanks, I have moved these (but I don't understand the rules for which go in `build:` and which go in `host:`?)

Reply:
The general idea is that anything linked against (zlib, libcurl, etc.) should go in the `host:` section, whereas things used purely for building the package should go in the `build:` section (compilers, preprocessing packages, make/cmake, etc.).